devices/Seesaw/SpanPinValuePair.cs

```diff
@@ -1,9 +1,12 @@
-namespace System
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.Seesaw
```

Comment:
where is this used? I don't think it is at all. That's related to the GPIO driver (which is not here and not useful) and I don't think that's needed.